misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -430,64 +444,91 @@ const SkeletonText = () => (
-        <MetadataMenu>
-          <Metadata label={"Content Types"} content={<SkeletonText />} />
-          <Metadata
-            label={"Application Interceptors"}
-            text={<SkeletonText />}
-          />
-          <Metadata label={"Network Interceptors"} content={<SkeletonText />} />
-          <Metadata label={"Send a Request"} content={<SkeletonText />} />
-        </MetadataMenu>
-      </Column>
-    </FlexContainer>
-  </Card>
+const SkeletonWebActions = (props: IState & IDispatchProps) => (
+  <div>
+    <FilterWebActions {...props} />
+    <Card>
+      <Header>
+        {[HTTPMethod.GET].map(m => (
+          <MethodTag method={m} />
+        ))}
+        <FloatLeft>
+          <H3 className={Classes.SKELETON}>{"AnotherSimpleWebAction"}</H3>
+        </FloatLeft>
+        <FloatLeft>
+          <CodeTag large={true}>{<SkeletonText />}</CodeTag>
+        </FloatLeft>
+      </Header>
+      <FlexContainer>
+        <Column>
+          <MetadataMenu>
+            <Metadata label={"Function"} content={<SkeletonText />} />
+            <Metadata label={"Services"} content={<SkeletonText />} />
+            <Metadata label={"Roles"} content={<SkeletonText />} />
+            <Metadata label={"Access"} content={<SkeletonText />} />
+          </MetadataMenu>
+        </Column>
+        <Column>
+          <MetadataMenu>
+            <Metadata label={"Content Types"} content={<SkeletonText />} />
+            <Metadata
+              label={"Application Interceptors"}
+              text={<SkeletonText />}
+            />
+            <Metadata
+              label={"Network Interceptors"}
+              content={<SkeletonText />}
+            />
+            <Metadata label={"Send a Request"} content={<SkeletonText />} />
+          </MetadataMenu>
+        </Column>
+      </FlexContainer>
+    </Card>
+  </div>
 )
 
 const WebActionsContainer = (props: IState & IDispatchProps) => {
   const metadata = simpleSelect(props.webActions, "metadata")
   if (metadata.length > 0) {
+    const filterKey =
+      WebActionInternal[
+        simpleSelect(props.simpleForm, `${filterTag}::HTMLSelect`, "data") ||
+          "All Metadata"
+      ]
+    const filterValue = simpleSelect(
+      props.simpleForm,
+      `${filterTag}::Input`,
+      "data"
+    )
+    const filteredMetadata = chain(metadata)
+      .filter((action: IWebActionInternal) =>
+        (action as any)[filterKey]
+          .toLowerCase()
+          .includes(filterValue.toLowerCase())
+      )
+      .value()
     return (
       <div>
-        <WebActions metadata={metadata} {...props} />
+        <FilterWebActions {...props} />
+        <WebActions
+          metadata={filteredMetadata as IWebActionInternal[]}
+          {...props}
+        />
       </div>
     )
   } else {
     // Displays mock of 5 Web Action cards which fill in when data is available
     return (
       <div>
-        <SkeletonWebActions />
+        <SkeletonWebActions {...props} />
```

Comment:
why do the mocks need this props reference?

Reply:
Initially had put in so the new filter form would be in the mock but realized I did it wrong so going to fix it and push.